pages/_document.js

```diff
@@ -0,0 +1,24 @@
+import Document, { Html, Head, Main, NextScript } from 'next/document';
+import React from 'react';
+
+class AppDocument extends Document {
+  render() {
+    return (
+      <Html lang="en">
+        <Head>
+          <meta charSet="utf-8" />
+          <meta name="viewport" content="width=device-width, initial-scale=1" />
+          <link href={'/public/images/logo.png'} rel="icon" type="image/png" />
```

Comment:
You can link to `/images/logo.png`

Next.js takes ur public folder and treats it like root!

Reply:
Yeah I think anything with a `public 
_url` you should be able to just remove! Also helps you from having to deal with that during production, which definitely won’t work with the base webpack plugins

Reply:
Great catch! I'll resolve;

> Why are u using document? What part of the html/body tag do you really need to target?

The first-party next eslint plugin [throws an error](https://nextjs.org/docs/messages/no-page-custom-font) when you try to import a font (or in general, anything that should span every single page) in `_app.js`. Seems like you're supposed to slap all of that stuff in `_document.js`, but I could be totally off base here; still new to things!

Reply:
Awww I see.. for fonts check out the comment I made above.. it should remove that necessity! 

As for importing things for every page, what I like to do (and what I have seen elsewhere) is basically to have a Layout component (which you already have) that takes care of all your headers!

https://github.com/UCLA-Creative-Labs/tears/blob/main/components/Layout.tsx

You can even create different Layout types and have those be utilized throughout ur app!